README.md

```diff
@@ -8,305 +8,409 @@
 
-To install the latest stable release of the operator, run the below script.
+Prefer to create the secrets and defaults yourself? See the [IBM Cloud Operator installation guide](docs/install.md).
 
-```bash
-curl -sL https://raw.githubusercontent.com/IBM/cloud-operators/master/hack/configure-operator.sh | bash -s -- install
-```
+### Using a service ID
 
-The above script stores an API key in a Kubernetes secret that can be accessed by the operator.
-Next, it sets default values used in provisioning IBM Cloud Services, like the resource group and region.
-You can override any default value in the `Service` custom resource.
-Finally, the script deploys the operator in your cluster.
+By default, the installation script creates an IBM Cloud API key that impersonates your user credentials, to use to set up the IBM Cloud Operator. However, you might want to create a service ID in IBM Cloud Identity and Access Managment (IAM). By using a service ID, you can control access for the IBM Cloud Operator without having the permissions tied to a particular user, such as if that user leaves the company. For more information, see the [IBM Cloud docs](https://cloud.ibm.com/docs/account?topic=account-serviceids).
 
-If you prefer to create the secret and the defaults manually, consult the [IBM Cloud Operator documentation](docs/install.md).
+1.  Create a service ID in IBM Cloud IAM. 
+    
+    ```bash
+    ibmcloud iam service-id-create serviceid-ico -d service-ID-for-ibm-cloud-operator
+    ```
 
-To install a specific version of the operator, you can pass a semantic version:
+2.  Assign the service ID access to the required permissions to work with the IBM Cloud services that you want the operator to manage. The required permissions vary with each IBM Cloud service. You can also scope an access policy to different regions or resource groups. For example, the following command grants the service ID the **Administrator** platform role in the default resource group in the US South (Dallas) region. For more information, see the [IBM Cloud docs](https://cloud.ibm.com/docs/account?topic=account-userroles).
+    
+    ```bash
+    ibmcloud iam service-policy-create serviceid-ico --roles Administrator --resource-group-name default --region us-south
+    ```
 
-```bash
-curl -sL https://raw.githubusercontent.com/IBM/cloud-operators/master/hack/configure-operator.sh | bash -s -- -v 0.0.0 install
-```
+3.  Create an API key for the service ID.
+    
+    ```bash
+    ibmcloud iam service-api-key-create apikey-ico serviceid-ico -d api-key-for-ibm-cloud-operator
+    ```
 
-### Uninstall
+4.  Set the API key of the service ID as your CLI environment variable. Now, when you run the installation script, the script uses the service ID's API key. The following command is an example for macOS.
+    
+    ```bash
+    setenv IBMCLOUD_API_KEY <apikey-ico-value>
```

Comment:
I think we need `export IBMCLOUD_API_KEY=` here instead of `setenv IBMCLOUD_API_KEY`. It looks like setenv is a thing in some shells, but not in Bash.